src/shared/lib/HotspotSet.spec.ts

```diff
@@ -44,8 +44,14 @@ describe('HotspotSet', () => {
     });
 
     it('works with one singleton region and one interval region', () => {
-        const hss = new HotspotSet([[-3, -3], [0, 5]]);
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki